library/entropy_poll.c

```diff
@@ -50,26 +50,42 @@
 
 #include <windows.h>
 #if _WIN32_WINNT >= 0x0501 /* _WIN32_WINNT_WINXP */
-#include <wincrypt.h>
+#include <bcrypt.h>
+#if defined(_MSC_VER) && _MSC_VER <= 1600
+/* Visual Studio 2010 and earlier issue a warning when both <stdint.h> and
+ * <intsafe.h> are included, as they redefine a number of <TYPE>_MAX constants.
+ * These constants are guaranteed to be the same, though, so we suppress the
+ * warning when including intsafe.h.
+ */
+#pragma warning( push )
+#pragma warning( disable : 4005 )
```

Comment:
We no longer put spaces after opening `(` and before closing `)`, so these should be

```
#pragma warning(push)
#pragma warning(disable : 4005)
```

(and elsewhere)

Reply:
Good catch. I run the fix_code_style script but it completely is ingoring PRAGMA directives.